ocp_utilities/exceptions.py

```diff
@@ -15,11 +18,11 @@ class NodesNotHealthyConditionError(Exception):
 
 
 class CommandExecFailed(Exception):
-    def __init__(self, name, err=None):
+    def __init__(self, name: str, err: str = ""):
```

Comment:
Should the return type be added for __init__ as well?

Reply:
yes; missed